docs/hearing/workflow/hearing-workflowConfig.json

```diff
@@ -93,7 +95,14 @@
           "actions": [
             {
               "action": "CLOSE",
-              "nextState": "COMPLETED",
+              "nextState": "HEARD",
+              "roles": [
+                "HEARING_CLOSER"
```

Comment:
@atulgupta2024 please review this once more as per our discussion yesterday and update.

Reply:
used HEARING_EDITOR